### PR TITLE
chore(sdk): native dependencies version updates

### PR DESCRIPTION
**Pull request by stefan-tudor**

### CODE REVIEW

**Review by eugenezhernakov — APPROVED** [review]

File: ios/Wrappers/RNWrappers.m [hunk 1]
```diff
@@ -412,8 +412,12 @@ + (JPRecommendationConfiguration *)recommendationConfigurationFromConfiguration:
     NSURL *URL = [NSURL URLWithString:URLString];
     NSString *RSAPublicKey = [dictionary stringForKey:@"rsaPublicKey"];
     NSNumber *timeout = [dictionary optionalNumberForKey:@"timeout"];
+    NSNumber *haltTransactionInCaseOfAnyError = [dictionary optionalBoolForKey:@"haltTransactionInCaseOfAnyError"];
```
Comment (RafalOzogJudopay) [inline ios/Wrappers/RNWrappers.m:416]: Is it necessary that we receive haltTransactionInCaseOfAnyError as NSNumber, instead of bool straight away?

**Review by RafalOzogJudopay — APPROVED** [review]
I left one comment in RNWrapper, please have a look. Approved

File: ios/Wrappers/RNWrappers.m (hunk 1 above)
Comment (stefan-tudor) [reply ios/Wrappers/RNWrappers.m:416]: That's an `NSDictionary`; you can only have Objects in dictionaries. Hence, that's why we have an `NSNumber` instance there.

Comment (RafalOzogJudopay) [reply ios/Wrappers/RNWrappers.m:416]: oh right, makes sense, good to know

**Pull request merged**